CHANGES.md

```diff
@@ -1,3 +1,7 @@
+### 6.12.1 (2018-08-31)
```

Comment:
Maintainers will update the changes file prior to a release, no need to do it in a PR.

Reply:
OK